test/libsolidity/SMTChecker.cpp

```diff
@@ -443,7 +443,7 @@ BOOST_AUTO_TEST_CASE(storage_value_vars)
 	string text = R"(
 		contract C
 		{
-			address a;
+			uint a;
```

Comment:
@leonardoalt @chriseth any reason this test case was using `address`?

Reply:
I think we wanted to test all supported types, so this actually should be `address`.